lists/ar.csv

```diff
@@ -0,0 +1,200 @@
+http://www.perspectivasur.com,NEWS,News Media,2017-07-04,ADC
+http://www.tribunadigital.com.ar,NEWS,News Media,2017-07-04,ADC
+https://taringa.net,MISC,Miscelaneous content,2017-07-04,ADC
+https://poringa.net,PORN,Pornography,2017-07-04,ADC
+http://www.launiondigital.com.ar,NEWS,News Media,2017-07-04,ADC
+http://musicaypropuestas.com.ar,NEWS,News Media,2017-07-04,ADC
+http://www.noticiasnet.com.ar,NEWS,News Media,2017-07-04,ADC
+http://www.atlantida.com.ar,NEWS,News Media,2017-07-04,ADC
+http://www.primeraedicion.com.ar,NEWS,News Media,2017-07-04,ADC
+http://www.p23.com.ar,NEWS,News Media,2017-07-04,ADC
+http://www.carasycaretas.org.ar,MISC,Miscelaneous content,2017-07-04,ADC
+http://www.elgrafico.com.ar,NEWS,News Media,2017-07-04,ADC
+http://www.gente.com.ar,NEWS,News Media,2017-07-04,ADC
+http://www.pcr.org.ar,POLR,Political Criticism,2017-07-04,ADC
+http://www.perfil.com,NEWS,News Media,2017-07-04,ADC
+http://www.revistabarcelona.com.ar,POLR,Political Criticism,2017-07-04,ADC
+http://www.revistaenie.clarin.com,NEWS,News Media,2017-07-04,ADC
+http://www.revista-noticias.com.ar,NEWS,News Media,2017-07-04,ADC
+http://www.revistanah.com.ar,MISC,Miscelaneous content,2017-07-04,ADC
+http://www.rollingstone.com.ar,NEWS,News Media,2017-07-04,ADC
+http://www.revistathc.com,ALDR,Alcohol & Drugs,2017-07-04,ADC
+https://revisbarcelona.com,POLR,Political Criticism,2017-07-04,ADC
+http://www.eldia.com,NEWS,News Media,2017-07-04,ADC
+http://edicionlimite.com.ar,NEWS,News Media,2017-07-04,ADC
+http://www.fopea.org,MISC,Miscelaneous content,2017-07-04,ADC
+http://www.minutorioja.com.ar,NEWS,News Media,2017-07-04,ADC
+http://opisantacruz.com.ar,NEWS,News Media,2017-07-04,ADC
+http://www.infonews.com,NEWS,News Media,2017-07-04,ADC
+http://www.tiempoar.com.ar,NEWS,News Media,2017-07-04,ADC
+http://www.veintitres.com.ar,NEWS,News Media,2017-07-04,ADC
+http://www.buenosairesherald.com,NEWS,News Media,2017-07-04,ADC
+http://www.ambito.com,NEWS,News Media,2017-07-04,ADC
+http://www.diario26.com,NEWS,News Media,2017-07-04,ADC
+http://www.elnueve.com.ar,NEWS,News Media,2017-07-04,ADC
+http://www.diarioveloz.com,NEWS,News Media,2017-07-04,ADC
+http://www.infojusnoticias.gov.ar,GOVT,Government,2017-07-04,ADC
+http://lmdiario.com.ar,NEWS,News Media,2017-07-04,ADC
+http://www.lanueva.com,NEWS,News Media,2017-07-04,ADC
+http://madres.org,MISC,Miscelaneous content,2017-07-04,ADC
+http://www.riojalibre.com.ar,NEWS,News Media,2017-07-04,ADC
+http://www.laizquierdadiario.com,NEWS,News Media,2017-07-04,ADC
+http://partidopirata.com.ar,POLR,Political Criticism,2017-07-04,ADC
+http://www.mapuche.info,NEWS,News Media,2017-07-04,ADC
+http://www.elorejiverde.com,NEWS,News Media,2017-07-04,ADC
+http://www.jachallaopinion.com.ar,NEWS,News Media,2017-07-04,ADC
+http://www.jachalmagazine.com.ar,NEWS,News Media,2017-07-04,ADC
+https://vimeo.com/205117580,MISC,Miscelaneous content,2017-07-04,ADC
+http://www.cha.org.ar,MISC,Miscelaneous content,2017-07-04,ADC
```

Comment:
It would be best to limit the usage of the MISC category code. As a matter of fact the MISC category code was added just to be able to support mapping legacy codes to the new scheme, it's usage for new test lists is highly discouraged.